tests/Service/SiteServiceTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Tests\Service;
+
+use App\Entity\Site;
+use App\Service\SiteService;
+use Doctrine\ORM\EntityManagerInterface;
+
+class SiteServiceTest extends ServiceTestCase
+{
+    protected $service;
+    protected $id;
+
+    public function testCaborConsentDisplay(): void
```

Comment:
After inverting the logic (see comment above), let's also make sure we are testing both scenarios - a site in CA and a site not in CA.